port/raspberrypi/rp2xxx/rp2040_ram_image.ld

```diff
@@ -0,0 +1,52 @@
+/*
+ * Target CPU:  ARM Cortex-M0+
+ * Target Chip: RP2040
+ */
+
+ENTRY(_entry_point);
```

Comment:
This doesn't work! The elf entry point is incorrect, but the bootloader doesn't care and jumps to the lowest address flashed, which is 0x20000000

Reply:
Fixed this by setting it in the target via 8d07e064 and 69733398